src/test/java/dev/harrel/jsonschema/ValidatorResultTest.java

```diff
@@ -9,6 +9,104 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 class ValidatorResultTest {
+
+    @Test
+    void returnsErrorMessageWhenAnyOfFails() {
```

Comment:
Could you also add test for `IfThenElse`?

Reply:
Thanks, missed that one but should be good now